lib/Twig/Extension/PimcoreObjectExtension.php

```diff
@@ -61,4 +64,19 @@ public function getFieldDefinitionFromJson(array|string $definition, string $typ
 
         return DataObject\Classificationstore\Service::getFieldDefinitionFromJson($definition, $type);
     }
+
+    public function getPimcoreElementTags(?AbstractElement $element, bool $asNameList): array
+    {
+        if (!$element) {
+            return [];
+        }
```

Comment:
```suggestion
    public function getPimcoreElementTags(AbstractElement $element, bool $asNameList): array
    {
       
```
Is there a specific reason why element can be null? Does it make sense to call that function without element, since it just returns an empty array in that case?

Reply:
Fine for now, follow up will address new method/improvements